Dockerfile-linux.template

```
@@ -259,6 +259,7 @@ RUN set -eux; \
 			"openssl-dev",
 			"readline-dev",
 			"sqlite-dev",
+			"gnu-libiconv-dev",
```

This top list is in sorted order (so this should be right after `curl-dev` above); I think it'd also be useful to point to https://www.php.net/manual/en/intro.iconv.php as the "upstream recommendation" to use this, ala:

```suggestion
			"gnu-libiconv-dev", # https://www.php.net/manual/en/intro.iconv.php "it'd be a good idea to install the GNU libiconv library"
```